assert.js

```diff
@@ -0,0 +1,21 @@
+/**
+  * Ensures an expected string is not empty.
+  *
+  * @param {Object} options - Options to use.
+  * @param {string} options.value - The expected string.
+  * @param {string} options.key - The identifier for the parameter.
+  *
+  * @throws - If the value is not a string or is empty.
+  * @returns {undefined} - It just throws or you are ok.
+*/
+export function notEmptyString({value, key}) {
```

Comment:
We're asserting a "non-empty string" not that the value is "not an empty string":

```suggestion
export function nonEmptyString({value, key}) {
```

But +1 to mirroring the assert-plus syntax to keep things simple/isomorphic as much as possible. That would also allow for a future easier lifting of this stuff into a frontend-compatible assertion lib.